src/privileged/blurts/subscripts/PanelUI.jsm

```diff
@@ -0,0 +1,138 @@
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.className = "redText";
+    elt.appendChild(doc.createTextNode("Compromised Accounts"));
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    this.pwnCountElt = elt;
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.className = "redText";
+    elt.appendChild(doc.createTextNode("Compromised Data"));
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt.className = "bottomBorder";
+    this.breachDataElt = elt;
+    box.appendChild(elt);
+
+    elt = doc.createElementNS(XUL_NS, "description");
+    elt2 = doc.createElementNS(HTML_NS, "span");
+      elt2.appendChild(doc.createTextNode("This website was reported to "));
+
+      elt3 = doc.createElementNS(HTML_NS, "a");
+        elt3.addEventListener("click", (event) => {
+          event.preventDefault();
+          doc.defaultView.openTrustedLinkIn(`https://monitor.firefox.com/?breach=${this.site.Name}`, "tab", {});
+        });
+        elt3.appendChild(doc.createTextNode("Firefox Monitor"));
+        this.monitorLink = elt3;
+      elt2.appendChild(elt3);
+
+      elt2.appendChild(doc.createTextNode(", a service that collects information about data breaches and other ways hackers can steal your information."));
+    elt.appendChild(elt2);
+    elt.appendChild(doc.createTextNode("This website was reported to Firefox Monitor, a service that collects information about data breaches and other ways hackers can steal your information."));
+    elt.className = "specialStuff";
+    box.appendChild(elt);
+
+    this.box = box;
+  }
+
+  get FirefoxMonitorUtils() {
+    return this.doc.defaultView.FirefoxMonitorUtils;
+  }
+
+  get primaryAction() {
+    return {
+      label: "Go to Firefox Monitor",
+      accessKey: "f",
+      callback: () => {
+        this.doc.defaultView.openTrustedLinkIn(`https://monitor.firefox.com/?breach=${this.site.Name}`, "tab", { });
+      },
+    };
+  }
+
+  get secondaryActions() {
+    return [
+      {
+        label: "Dismiss",
+        accessKey: "d",
+        callback: () => { },
+      }, {
+        label: "Never show breach alerts",
+        accessKey: "n",
+        callback: () => {
+          this.FirefoxMonitorUtils.disableBlurts();
+        },
+      },
+    ];
+  }
+
+  refresh(site) {
+    this.site = site;
+
+    const doc = this.doc;
+
+    this.logoElt.style.backgroundImage = `url(${this.FirefoxMonitorUtils.getURL(`PwnedLogos/${site.logoSrc}`)}`;
+
+    function clearChildren(elt) {
+      while (elt.firstChild) {
+        elt.firstChild.remove();
+      }
+    }
+
+    for (const elt of [this.breachNameElt, this.breachDateElt, this.pwnCountElt, this.breachDataElt]) {
+      clearChildren(elt);
+    }
+
+    this.breachNameElt.appendChild(doc.createTextNode(site.Name));
+
+    this.breachDateElt.appendChild(doc.createTextNode(site.BreachDate));
```

Comment:
Not sure if we want to format the `BreachDate` here (to match the server behavior).

https://github.com/mozilla/blurts-server/blob/bc5dd0f2fe3ef6c804a3b9ff94ac1b0b457f2947/views/partials/banner_breach.hbs#L9

https://github.com/mozilla/blurts-server/blob/d963172fdc5fe3b5dc5d2a02977e5c109715d573/hbs-helpers.js#L6-L9